docs/unified-test-documentation/dasharo-compatibility/341-pfSense-support.md

```diff
@@ -0,0 +1,114 @@
+# Dasharo Compatibility: pfSense support
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+## PFS001.001 pfSense serial installation
```

Comment:
I think it will be good to change test case names. Maybe something like this:

```suggestion
## PFS001.001 pfSense stable (serial output) installation on Hard Disk
```